# The optimal motion plan represented in cartesian space

 HyqStateEE[] states    # trajectory of states
